Tests/Configuration/ConfigurationBuilderTest.php

```diff
@@ -34,29 +31,29 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->container = new Container();
-        $this->container->addScope(new Scope('request'));
-        $this->container->enterScope('request');
+        $this->container = $this->getMockBuilder('Symfony\Component\DependencyInjection\Container')
```

Comment:
I think this can stay as `$this->container = new Container();`, it doesn't need to be mocked